server/services/gamecontrol.go

```diff
@@ -0,0 +1,37 @@
+package services
+
+import (
+	"context"
+
+	controller "github.com/MettyS/checkers/server/controller"
+	pb "github.com/MettyS/checkers/server/generated"
+)
+
+// GameControlServer server for game control service
+type GameControlServer struct {
+	pb.UnimplementedGameControlServiceServer
+}
+
+func convertGameStartRequestInward(req *pb.GameStartRequest) *controller.GameStartRequest {
```

Comment:
Why don't these functions return values directly, rather than refs/pointers?  It seems like values would be better if possible.

Reply:
I refactored the services and controller to have internal-level avoid references. This is a good idea.